graphite-clickhouse.go

```diff
@@ -59,7 +60,7 @@ func WrapResponseWriter(w http.ResponseWriter) *LogResponseWriter {
 	return &LogResponseWriter{ResponseWriter: w}
 }
 
-func Handler(handler http.Handler) http.Handler {
+func Handler(handler http.Handler, cfg *config.Config) http.Handler {
```

Comment:
Isn't it the only place to change? Why all other handlers (capabilities, find, etc..) are changed too? It should duplicate the logging then.